src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/datetimechecker/SummaryReportCronAwareChecker.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.service.datetimechecker;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.support.CronSequenceGenerator;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import static java.util.TimeZone.getTimeZone;
+import static uk.gov.hmcts.reform.blobrouter.util.TimeZones.EUROPE_LONDON_ZONE_ID;
+
+/**
+ * Implementation of {@link DateRelevantForCurrentReconciliationChecker}, which checks time relevancy based on defined
+ * cron's expression.
+ */
+@Component
+public class SummaryReportCronAwareChecker implements DateRelevantForCurrentReconciliationChecker {
+    private final String cron;
+
+    public SummaryReportCronAwareChecker(
+        @Value("${scheduling.task.create-reconciliation-summary-report.cron}")
+        String cron) {
```

Comment:
we put `)` in a new line (althought sadly it's not checked automatically)

Reply:
Done.